mrc/control/steering/exceptions.py

```diff
@@ -0,0 +1,12 @@
+class SteeringException(Exception):
```

Comment:
In mrc/shared/exceptions is an MRCException which should be used to derive all exceptions. Also I think we agreed to place all exceptions in that place.